src/registries/menus/number_format_menu_registry.ts

```diff
@@ -1,9 +1,12 @@
-import { ActionSpec } from "../../actions/action";
+import { SpreadsheetChildEnv } from "../..";
+import { ActionSpec, createActions } from "../../actions/action";
 import * as ACTION_FORMAT from "../../actions/format_actions";
+import { isDateTimeFormat, memoize } from "../../helpers";
 import { _t } from "../../translation";
-import { MenuItemRegistry } from "../menu_items_registry";
+import { Format } from "../../types";
+import { Registry } from "../registry";
 
-export const numberFormatMenuRegistry = new MenuItemRegistry();
+export const numberFormatMenuRegistry = new Registry<ACTION_FORMAT.NumberFormatActionSpec>();
```

Comment:
Is this registry useful BTW ? We never extend it, or use it outside of this file. Shoudl this just be an array ?

Reply:
I don't think it's very harmful to keep it as it is for now. We never know what kind of feature we might want to introduce in Odoo